### PR TITLE
Adding the new map layouts

### PR DESCRIPTION
**Pull request by narendasan**
Adds the 3 map layouts proposed in slack to the environment.

### CODE REVIEW

**Comment by narendasan** [comment]
Not sure how to test these with just a human agent locally. Is there a command I can run to do so?

**Comment by Jmiceli17** [comment]
> Not sure how to test these with just a human agent locally. Is there a command I can run to do so?

Yep! Clone and setup the [multiHRI repo](https://github.com/HIRO-group/multiHRI) and then run:
```
python scripts/run_overcooked_game.py
```

Some things to note:

- By default, [run_overcooked_game.py](https://github.com/HIRO-group/multiHRI/blob/main/scripts/run_overcooked_game.py) will load a trained FCP agent for 2 of the chefs, and allow you to play as the third (blue hat) chef. 

- Make sure you download [this folder](https://o365coloradoedu-my.sharepoint.com/personal/avab4907_colorado_edu/_layouts/15/onedrive.aspx?csf=1&web=1&e=2ZXvGz&CID=e130e83e%2Dd98c%2D441d%2Daf0a%2Db05f344a22cc&id=%2Fpersonal%2Favab4907%5Fcolorado%5Fedu%2FDocuments%2FResearch%2FShared%2FZero%2DShot%20Coordination%2Fagent%5Fmodels&FolderCTID=0x012000C5E2691128F1E049943EE85868D9B09A&view=0) and save it at `multiHRI/agent_models`. This folder has pre-trained agents in it that we can use for evaluation during experiments

- [These are the lines](https://github.com/HIRO-group/multiHRI/blob/main/scripts/run_overcooked_game.py#L30-L31) that specify the path to the FCP agent that gets loaded for the game (that they assume you have performed the step above)

- If you want to test your layouts, just change the string on [this line](https://github.com/HIRO-group/multiHRI/blob/main/scripts/run_overcooked_game.py#L27) to be the name of the layout you want to test.

- If you're running in WSL, you will need to have an [X-server (or similar)](https://sourceforge.net/projects/vcxsrv/) running so the pygame window can open

File: src/overcooked_ai_py/data/layouts/long_counter_circuit.layout
Comment (Jmiceli17) [inline src/overcooked_ai_py/data/layouts/long_counter_circuit.layout:1]: to conform to the naming format on other layouts rename this to be `3_chefs_long_counter_circuit.layout`

File: src/overcooked_ai_py/data/layouts/long_onion.layout
Comment (Jmiceli17) [inline src/overcooked_ai_py/data/layouts/long_onion.layout:1]: to conform to the naming format on other layouts rename this to be `3_chefs_long_onion.layout`

File: src/overcooked_ai_py/data/layouts/unequal_kitchens.layout
Comment (Jmiceli17) [inline src/overcooked_ai_py/data/layouts/unequal_kitchens.layout:1]: to conform to the naming format on other layouts rename this to be `3_chefs_unequal_kitchens.layout`

**Review by Jmiceli17 — COMMENTED** [review]
Looks good! Just rename the files and check that you can actually run them

**Comment by narendasan** [comment]
I'm testing the environments and it seems like with the long onion, if you place an empty plate next to the pot (img below), the game crashes with 

```sh
Traceback (most recent call last):
  File "scripts/run_overcooked_game.py", line 42, in <module>
    dc.on_execute()
  File "/home/naren/Developer/py/mhri/multiHRI/oai_agents/common/overcooked_gui.py", line 274, in on_execute
    done = self.step_env(action)
  File "/home/naren/Developer/py/mhri/multiHRI/oai_agents/common/overcooked_gui.py", line 194, in step_env
    obs, reward, done, info = self.env.step(agent_action)
  File "/home/naren/Developer/py/mhri/multiHRI/oai_agents/gym_environments/base_overcooked_env.py", line 232, in step
    tm_obs = self.get_obs(c_idx=t_idx, enc_fn=teammate.encoding_fn)
  File "/home/naren/Developer/py/mhri/multiHRI/oai_agents/gym_environments/base_overcooked_env.py", line 199, in get_obs
    obs['subtask_mask'] = self.action_masks(c_idx)
  File "/home/naren/Developer/py/mhri/multiHRI/oai_agents/gym_environments/base_overcooked_env.py", line 182, in action_masks
    self.valid_counters, USEABLE_COUNTERS.get(self.layout_name, 5)).astype(bool)
  File "/home/naren/Developer/py/mhri/multiHRI/oai_agents/common/subtasks.py", line 156, in get_doable_subtasks
    elif obj.name == 'dish' and prev_subtask != 'put_plate_closer' and obj.position in valid_counters[p_idx]:
IndexError: list index out of range
```


image

**Comment by avv-va** [comment]
This is okay just comment that line that causes the crash (its Stephane's code that we don’t use)

**Comment by narendasan** [comment]
Should I push up commenting out those lines to the mutliHRI repo?

**Comment by avv-va** [comment]
No, it’s fine

**Pull request merged**